indico/modules/events/papers/models/call_for_papers.py

```diff
@@ -89,10 +91,19 @@ def layout_reviewers(self):
                 if p.has_management_role('paper_layout_reviewer', explicit=True)}
 
     @property
-    def acl_entries(self):
-        principals = self.managers | self.judges
+    def assignees(self):
+        users = self.judges
         if self.content_reviewing_enabled:
-            principals = principals | self.content_reviewers
+            users |= self.content_reviewers
         if self.layout_reviewing_enabled:
-            principals = principals | self.layout_reviewers
-        return principals
+            users |= self.layout_reviewers
+        return users
+
+    @property
+    @memoize_request
+    def user_competences(self):
+        user_ids = {user.id for user in self.event.cfp.assignees}
+        user_competences = (PaperCompetence.query.with_parent(self.event)
+                            .filter(PaperCompetence.user_id.in_(user_ids))
+                            .all())
+        return {competences.user_id: competences for competences in user_competences}
```

Comment:
`competence` instead of not `competences`? (better fits the model name). Or just a generic loop var like `x` or `c` ;)